### PR TITLE
Replace bazel with Makefile and bash

### PR DESCRIPTION
**Pull request by qinqon**
The gocli project is not complex enough to require bazel

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
/check-provision-k8s-1.18

File: cluster-provision/gocli/hack/install-go.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -xe
+
+destination=$1
```
Comment (rmohr) [inline cluster-provision/gocli/hack/install-go.sh:3]: Why do we need this script?

Comment (qinqon) [reply cluster-provision/gocli/hack/install-go.sh:3]: Is a simplified version of go gimme from travis, it install go under `build` directory so we vendor the go compiler.

Comment (rmohr) [reply cluster-provision/gocli/hack/install-go.sh:3]: Why do we not use an image where go is installed?

Comment (qinqon) [reply cluster-provision/gocli/hack/install-go.sh:3]: The idea is to post gocli binaries at release page at github no gocli container will exists, so we do a plain compilation with a vendored go.

**Comment by qinqon** [comment]
/hold

Prow does not like it.

**Pull request merged**